lib/Ast.ml

```diff
@@ -66,8 +36,14 @@ and color =
   | BrightCyan
   | BrightWhite
 
+(** [default_text_color] : Default Text Color *)
+let default_text_color = Black
+
+(** [default_background_color] : Default Background Color *)
+let default_background_color = White
```

Comment:
Peut être rajouté une nouvelle pseudo couleur qui correspondrait à la couleur par défaut ? C'est le cas avec les séquences VT 100.

Reply:
Argl ! On 4 bits pour encoder les couleurs :/ Du coup un 5e bit pour rajouter celle-ci et d'autre ? (On a largement la place)

Reply:
Pour moi, ça me va

Reply:
Fixed in 1a1d91b477b5fef9fbb93878e73e4fca242784d0.